extension/main.js

```diff
@@ -13,6 +13,15 @@
 		chrome.browserAction.setTitle({
 			title: title
 		});
+
+		if (badge && window.GitHubNotify.settings.get('showDesktopNotif')) {
+			chrome.notifications.create('github-notifier', {
+				title: 'GitHub',
+				iconUrl: 'icon-128.png',
+				type: 'basic',
+				message: 'You have new unread notifications'
```

Comment:
Ok, let's ignore the buttons. Though I still feel it would be much more useful if you could see the notification title. Then I would know if it's something I care about or not, instead of just being notified for random unimportant notifications.